Source/WebKit/WebProcess/WebPage/DrawingArea.messages.in

```diff
@@ -22,8 +22,9 @@
 
 messages -> DrawingArea NotRefCounted {
 #if USE(COORDINATED_GRAPHICS) || USE(TEXTURE_MAPPER)
-    UpdateBackingStoreState(uint64_t backingStoreStateID, bool respondImmediately, float deviceScaleFactor, WebCore::IntSize size, WebCore::IntSize scrollOffset)
+    UpdateGeometry(WebCore::IntSize size) -> ()
```

Comment:
WinCairo EWS failed. WinCairo is already using `UpdateGeometry`. Could you rename either message?